packages/connect/src/api/changeLanguage.ts

```diff
@@ -54,64 +42,12 @@ export default class ChangeLanguage extends AbstractMethod<'changeLanguage', Cha
         return uiResp.payload;
     }
 
-    private async uploadTranslationData(payload: ArrayBuffer | null) {
-        if (!this.device.commands) {
-            throw ERRORS.TypedError('Runtime', 'uploadTranslationData: device.commands is not set');
-        }
-
-        if (payload === null) {
-            const response = await this.device.commands.typedCall(
-                'ChangeLanguage',
-                ['Success'],
-                { data_length: 0 }, // For en-US where we just send `ChangeLanguage(size=0)`
-            );
-
-            return response.message;
-        }
-
-        const length = payload.byteLength;
-
-        let response = await this.device.commands.typedCall(
-            'ChangeLanguage',
-            ['TranslationDataRequest', 'Success'],
-            { data_length: length },
-        );
-
-        while (response.type !== 'Success') {
-            const start = response.message.data_offset!;
-            const end = response.message.data_offset! + response.message.data_length!;
-            const chunk = payload.slice(start, end);
-
-            response = await this.device.commands.typedCall(
-                'TranslationDataAck',
-                ['TranslationDataRequest', 'Success'],
-                {
-                    data_chunk: Buffer.from(chunk).toString('hex'),
-                },
-            );
-        }
-
-        return response.message;
-    }
-
     async run() {
```

Comment:
`async` is redundant here? Detected by linter.

Reply:
b71f715faa69236afc5cccc8db12a375c2b5ac06